engine/src/main/java/org/destinationsol/game/ObjectManager.java

```diff
@@ -189,7 +192,7 @@ private void removeObjNow(SolGame game, SolObject o) {
         myObjs.remove(o);
         myRadii.remove(o);
         o.onRemove(game);
-        game.getDrawableManager().removeObject(o);
+        game.getContext().get(DrawableManager.class).removeObject(o);
```

Comment:
Is there any reason you use `game.getContext()` here rather than `myContext`?

Reply:
Nope - that was a mistake. Fixing it now